app/helpers/analytics_helper.rb

```diff
@@ -1,4 +1,13 @@
 # frozen_string_literal: true
 
 module AnalyticsHelper
+  def case_updates
+    # returns all versions of all cases created in the last 30 days
+    PaperTrail::Version.where('created_at > ?', 30.days.ago)
+  end
+
+  def case_updaters_count(how_many)
+    # returns list of uniq editors of case updates along woth thenumber of updates each has authored
+    case_updates.group(:whodunnit).order('count_id DESC').limit(how_many).count(:id)
```

Comment:
Is this going to return the actual user objects?  Spot check makes me think this returns 10, a count of how many values in the group are returned, which is limited based on the parameter. 

Either way, it would be good to return as little info as necessary - below it uses the name and a link to the user path.  Let's see if we can use something like `pluck` to grab the user name and construct the URL.

Reply:
This function takes the population of all case versions created in the past 30 days and returns a list of `how_many' arrays each with two elements: `editor_id` and `number_of_updates`, where `editor_id` is the user_id of the editor responsible for creating a case version and `number_of_updates is the quantity of versions that editor created. The list returned is ordered by the number of updates from most to least.

I have to investigate `pluck`. I have never used that before.

Reply:
Nah, I misspoke - I thought it was returning the whole object.  Two values is perfectly fine in my opinion - just what we need and nothing more.  If we were returning whole objects when we only needed the values for two columns, then we could have used `pluck` to improve performance.